l10n_it_invoices_data_communication/models/communication.py

```diff
@@ -41,12 +41,21 @@ def check_normalized_string(value):
     return normalized
 
 
+# compiled reg expression for whitespace characters substitution
+reg_whitespace = re.compile(r'\s+')
+
+
 class ComunicazioneDatiIva(models.Model):
     _name = 'comunicazione.dati.iva'
     _description = 'Invoices data communication'
     _rec_name = 'identificativo'
     _inherit = ['mail.thread']
 
+    @staticmethod
+    def encode(string_to_encode, max_chars, encoding='latin'):
```

Comment:
Invece di usare un staticmethod nella classe non possiamo definire la funzione fuori dalla classe così che possa essere richiamata, all'evenienza, anche da altre parti importandola?

Reply:
Io farei una bella classe ``AbstractModel`` con le funzionalità da ereditare senza aver bisogno di import

Reply:
@robyf70 da ereditare sia qui che in `l10n_it_fatturapa_out`? La mettiamo in `l10n_it_account`?

Reply:
> La mettiamo in l10n_it_account?

LGTM! 🚀

Reply:
Ho visto che questo modulo eredita da alcuni moduli (l10n_it_fiscal_document_type, l10n_it_account_tax_kind)  che a loro volta ereditano da l10n_it_account. 
Nel caso si decida di mettere la funzione in l10n_it_account, credo sia meglio mettere la dipendenza esplicita anche qui.

Reply:
Fatto